### PR TITLE
Default value should be empty dict literal string

### PR DESCRIPTION
**Pull request by caesar2164**
- student view uses empty dict if actual value is missing

### CODE REVIEW

**Comment by caesar2164** [comment]
this preempts a possible bug as mentioned here: https://github.com/Stanford-Online/xblock-in-video-quiz/pull/8

File: invideoquiz/invideoquiz.py [hunk 1]
```diff
@@ -71,6 +71,8 @@ def student_view(self, context=None):  # pylint: disable=unused-argument
         """
         Show to students when viewing courses
         """
+        timemap = self.timemap or {}
+
```
Comment (stvstnfrd) [inline invideoquiz/invideoquiz.py]: Nuke newline.

File: invideoquiz/invideoquiz.py [hunk 2]
```diff
@@ -48,7 +48,7 @@ class InVideoQuizXBlock(StudioEditableXBlockMixin, XBlock):
 
     timemap = String(
         display_name=_('Problem Timemap'),
-        default={},
+        default='',
```
Comment (stvstnfrd) [inline invideoquiz/invideoquiz.py]: What is the error message if this is `None`?

Comment (caesar2164) [reply invideoquiz/invideoquiz.py]: Because this is a string field, it it actually just displays as the string "None"

File: setup.py [hunk 3]
```diff
@@ -45,7 +45,7 @@ def run_tests(self):
 
 setup(
     name='invideoquiz-xblock',
-    version='0.2.0',
+    version='0.2.1',
```
Comment (stvstnfrd) [inline setup.py]: Why did we bump the minor version to x.2.x?

Per SemVer [1], the 3rd "variable" is the "patch" number, used to indicate a bugfix.
The "minor" number, what went from 1 to 2, is for backwards compatible feature changes.

I suspect this should just be `0.1.4`.

[1] http://semver.org/

**Comment by stvstnfrd** [comment]
Pleeease don't squash PRs mid-review;
the "remove P tag" should have been a separate commit.

**Comment by stvstnfrd** [comment]
+1

**Pull request merged**